src/sentry/models/rollbackorganization.py

```diff
@@ -19,3 +20,6 @@ class RollbackOrganization(DefaultFieldsModel):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_rollbackorganization"
+        constraints = [
```

Comment:
Since this is a single column, you can also set `unique=True` on the `FlexibleForeignKey`